CruzHacks/cruzhacks-2021-website
# Boilerplate background component layers

Problem
=======
As an engineer, I want a baseline background for cross-sectional components.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199332791703155)

Solution
========

I implemented a Background component that creates an overarching background for the sections. There is a separate background for Sections 1-4 and a separate background for Sections 5-7. Sections 1-4 have the basic background with a blue wire frame and Sections 5-7 have a gradient background with a rainbow wire frame.

Change summary:
---------------
* Added background component and imported new images
* added background to the home component


Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] `.env` & `.env.sample` 
- [ ] `webpack.config.js` 
- [ ] GCP Secret Manager
- [ ] Github Secrets 
- [ ] GithubActions Workflow

Steps to Verify:
----------------
1. Start the webserver to see the feature
2. It looks nice with only seven sections (4 in section 1 and 3 in section 2)


Screenshots (optional):
-----------------------
Show-n-tell images/animations here
